src/BTVNanoCommissioning/utils/plot_utils.py

```diff
@@ -0,0 +1,334 @@
+import matplotlib.pyplot as plt
```

Comment:
I understand this is copied and adjusted from the deprecated coffea hist class. Can the native plotratio function of matplotlib not be used?

Reply:
Indeed the plotratio function in `hist` class did not provide the correct error calculation which is suggested by @andrzejnovak